File: Yuva/angular/emp-app/Employee.ts
import iOperations from './Operations';
import Person from './Person';
export class Employee extends Person implements iOperations{
    private empId:number;
    private unit:string;
    protected projName:string;
    public setEmpId(eid:number){
        this.empId=eid;
    }
    public setUnit(unit:string){
        this.unit=unit;
    }
    public setprojName(proj:string){
        this.projName=proj;
    }
    public cabin(cabin:string){
        return cabin;
    }
    public role(role:string){
        return role;
    }
    public toString(){
        console.log("EmpID:"+this.empId+" <> "+this.unit+" <> "+" ProjName: "+" <> "+this.projName+" <> "+"Cabin:"+this.cabin+" <> "+"Role:"+this.role);
    }
}

File: Yuva/spring/sts-4.0/AopDemo/src/main/java/com/example/model/Employee.java
package com.example.model;

public class Employee {
	String name;
	String empid;
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getEmpid() {
		return empid;
	}
	public void setEmpid(String empid) {
		this.empid = empid;
	}
	@Override
	public String toString() {
		return "Employee [name=" + name + ", empid=" + empid + "]";
	}
	
}
